src/pages/video-details/video-container/VideoDetailsMain.jsx

```diff
@@ -29,6 +29,42 @@ const VideoDetailsMain = observer(({
   const [openedShareModal, {open: openModal, close: closeModal}] = useDisclosure(false);
   const [loadingSummary, setLoadingSummary] = useState(false);
   const [summary, setSummary] = useState(null);
+  const [currentThumb, setCurrentThumb] = useState(null);
+
+  const searchTerm = searchStore.currentSearch.terms;
+  const indexId = "iq__idx_unknown"; // XXX
```

Comment:
This just needs to be the currently selected index, right--`searchStore.currentSearch.index`?